### PR TITLE
value substitute attribute + bugfix

### PR DESCRIPTION
**Pull request by anadoba**

### CODE REVIEW

File: postman-generator/src/main/scala/generator/PostmanItemBuilder.scala [hunk 1]
```diff
@@ -109,8 +110,8 @@ object PostmanItemBuilder extends Logging {
   private def generatePathParamValue(parameter: Parameter): String = {
     val defaults = List(parameter.example, parameter.default).flatten
     defaults.headOption.getOrElse {
-      // Creating Postman Variable
-      s"{{${parameter.name.toUpperCase}}}"
+      // Referencing Postman Variable
+      s"{{${parameter.name}}}"
```
Comment (anadoba) [inline postman-generator/src/main/scala/generator/PostmanItemBuilder.scala:114]: variables are set with lower case

File: postman-generator/src/main/scala/models/attributes/PostmanAttributes.scala [hunk 2]
```diff
@@ -56,5 +57,6 @@ object PostmanAttributes {
     password: String
   )
 
+  implicit val config = JsonConfiguration(SnakeCase)
```
Comment (anadoba) [inline postman-generator/src/main/scala/models/attributes/PostmanAttributes.scala:60]: bugfix - sometimes in the algorithm we read `ExtendedObjectReference` so it needs to be snake case as the rest of ApiBuilder

File: postman-generator/src/test/resources/postman/expected-1.json [hunk 3]
```diff
@@ -222,7 +246,7 @@
             ],
             "body": {
               "mode": "raw",
-              "raw": "{\n  \"catalog_id\" : \"lorem ipsum 691503\"\n}"
+              "raw": "{\n  \"catalog_id\" : \"{{catalog#id}}\"\n}"
```
Comment (anadoba) [inline postman-generator/src/test/resources/postman/expected-1.json:249]: bugfix outcome - this part used `asOpt[ExtendedObjectReference]`

**Comment by ajanczak** [comment]
Lgtm

**Pull request merged**